### DISCUSSION TITLE
Audio files from Braw clips.

### DISCUSSION THREAD

**Discussion by Marks07-stack** [latenitefilms/BRAWToolbox#56] [Support] [answered]
Typically When I shoot braw I  have an onboard mic, Sanken CS1, on one channel and the seond channel is built in camera mono. I notice when using braw toolbox the resulting imported file in X seems to not have 2 individual channels. Typically I would deselect the built in camera mic in the inspector if this was a pro res file , but can’t seem to do this with an imported braw file. Is there something I am missing about separating the audio channels with an imported Braw clip?

**Comment by latenitefilms** [comment, marked as answer]
If you're creating Synchronised Clips, then you can "go inside" the Synchronised Clip and adjust the Audio Configuration.

If you're creating Multicam Clips, you'll need to "go inside" the Multicam Clip, THEN inside the Synchronised Clip to adjust.

I agree though - most of the time you'll always want to work with Dual Mono, instead of Stereo, so I'll look at adjusting this in a future update.

**Reply by Marks07-stack** [reply]
Thanks for the tip, I found what I needed. I was a bit thrown off by the fact that the audio controls I was looking for weren’t available in the inspector as per the usual .

**Reply by latenitefilms** [reply]
It's actually no different to a "normal" Synchronised Clip. In terms of audio, we don't do anything fancy - we literally just "rip out" the audio from the BRAW file as a WAV, then insert it into the Synchronised Clip. But yes, if you're not used to working with Synchronised Clips - which again, work a bit differently to Multicam Clips, it can be a bit confusing. Will try fix this soon!

**Comment by Marks07-stack** [comment]
Yes I work with sync clips - I frequently use track E or Sounddevices files in my edits, its just that normally my converted Braw clips travel with their audio or in  X or Resolve where everything is just a few more mouse clicks away.  I’ve been working in Resolve for the last few months and being able to work natively in X with  Braw is a breath fo fresh air. Thanks for cracking the code so we can use Braw with Final Cut.Mark SmithMobile: 917| 406| 6157On Feb 2, 2023, at 00:13, Chris Hocking ***@***.***> wrote:﻿
It's actually no different to a "normal" Synchronised Clip. In terms of audio, we don't do anything fancy - we literally just "rip out" the audio from the BRAW file as a WAV, then insert it into the Synchronised Clip. But yes, if you're not used to working with Synchronised Clips - which again, work a bit differently to Multicam Clips, it can be a bit confusing. Will try fix this soon!
